libs/schemas/src/commands/thread.schemas.ts

```diff
@@ -1,3 +1,21 @@
 import { z } from 'zod';
+import { Thread } from '../entities';
+import { DiscordMetaSchema, PG_INT } from '../utils';
 
-export const TodoThreadCommands = z.object({});
+export const CreateThread = {
+  input: z.object({
+    id: PG_INT,
```

Comment:
What would this id be?

Reply:
id is required in all commands, but it will be ignored internally

Reply:
I'm pretty sure we settled on making the ID optional in the last office hours @kurtisassad to confirm. Any command that creates a root object will require an `id` parameter that the client needs to set to some random arbitrary value. This is not a good pattern and quite frankly it introduces confusion when an id is required but actually isn't.

Reply:
I'm ok with making it optional, but we need another PR for this fix since it will affect more commands

Reply:
https://github.com/hicommonwealth/commonwealth/issues/9009